pkg/servicemesh/apis/servicemesh/v1/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Red Hat, Inc.
```

Comment:
instead of pulling in all the files under `pkg/servicemesh/apis` and `pkg/servicemesh/client`, we might want to try to use the client code in maistra/api. Another optional improvement :wink:

Reply:
Done in f580b7fb02